cdisc-org/cdisc-rules-engine
# Add operation to get variable's codelist from define

This PR adds an operation to get the codelist ccode for a specific variable from the provided define.xml. This is to support rules specified in #278.

Steps to test:

1. Run the unit tests
2. Run the following provided sample rule with the provided data and define xml:
[define.xml.txt](https://github.com/cdisc-org/cdisc-rules-engine/files/11499369/define.xml.txt)
[invalid_units_rule.json.txt](https://github.com/cdisc-org/cdisc-rules-engine/files/11499372/invalid_units_rule.json.txt)
[invalid_units_dataset.json.txt](https://github.com/cdisc-org/cdisc-rules-engine/files/11499375/invalid_units_dataset.json.txt)

using the command: `python core.py test -s sdtmig -v 3-4 -r invalid_units_rule.json -dp invalid_units_dataset.json`

3. Verify that the results are correct given the CCODE for PPORRESU in the define.
4. Modify the CCODE for PPORRESU and verify the results change accordingly
